lumigator/python/mzai/backend/backend/api/routes/runs.py

```diff
@@ -0,0 +1,61 @@
+"""These are the runs routes.
+A run is a single execution of an experiment, which can be a single job or a batch of jobs.
+"""
+
+from uuid import UUID
+
+from fastapi import APIRouter, BackgroundTasks, status
+from lumigator_schemas.experiments import (
+    ExperimentCreate,
+    ExperimentResponse,
+    ExperimentResultDownloadResponse,
+    ExperimentResultResponse,
+)
+from lumigator_schemas.extras import ListingResponse
+
+from backend.api.deps import ExperimentServiceDep, JobServiceDep
+
+router = APIRouter()
+
+
+@router.post("/", status_code=status.HTTP_201_CREATED)
+async def create_run(
+    service: ExperimentServiceDep, request: ExperimentCreate, background_tasks: BackgroundTasks
+) -> ExperimentResponse:
+    return ExperimentResponse.model_validate(service.create_experiment(request, background_tasks))
```

Comment:
What we currently call 'create-experiment` is really a merging of `create-experiment` (aka make new experiment id and store in db) and `trigger-run` (aka trigger inference job and then evaluation job)

We would want to only call something like `create_run` here, which would be a new component in the service layer.